File: app/models/Review.js
var restful = require('node-restful'),
    mongoose = restful.mongoose;

// MONGO SCHEMA
var review = restful.model('review', mongoose.Schema({
  title: {type:'string', required:true},
	reviewerName: {type:'string', required:true},
	score: {type:'Number', required:true},
	comment:{type:'string', required: false}
}));


exports = module.exports = review;

File: LmDataVacuum/core/mongoModels/review.py
from mongoModel import MongoModel

class Review(MongoModel):

    def __init__(self, title, reviewerName, score, comment):
        self.title = title
        self.reviewerName = reviewerName
        self.score = score
        self.comment = comment


    def exportJson(self):
        json = "{"
        json += "\"title\": \"" +self.title+"\","
        json += "\"reviewerName\": \"" +self.reviewerName+"\","
        json += "\"score\": " +self.score+","
        json += "\"comment\": \"" + self.comment+"\""
        json +="}"
        return json
